src/screens/SUE/SueHomeScreen.tsx

```diff
@@ -0,0 +1,141 @@
+  const {
+    staticContentName = 'staticContentList',
+    staticContentListDescription,
+    horizontal = true,
+    showStaticContentList = true,
+    staticContentListTitle
+  } = staticContentList;
+
+  const { data } = useStaticContent({
+    refreshTimeKey: `publicJsonFile-${staticContentName}`,
+    name: staticContentName,
+    type: 'json',
+    skip: !showStaticContentList
+  });
+
+  // function to add customized styles from `globalSettings` to the list items
+  const staticContentListItem = useMemo(() => {
+    if (!data) {
+      return [];
+    }
+
+    let listItem = data;
+
+    if (appDesignSystem?.staticContentList) {
+      listItem = listItem?.map((item: any) => ({
+        ...item,
+        appDesignSystem: appDesignSystem.staticContentList
+      }));
+    }
+
+    return listItem;
+  }, [appDesignSystem, data]);
+
+  return (
+    <SafeAreaViewFlex>
+      <ScrollView>
+        <ConnectedImagesCarousel
+          navigation={navigation}
+          publicJsonFile="sueHomeCarousel"
+          refreshTimeKey="publicJsonFile-sueHomeCarousel"
+        />
+
+        {!!sueReportListNavigationButton &&
+          sueReportListNavigationButton === LIST_NAVIGATION_BUTTON.TOP && (
+            <Wrapper noPaddingBottom style={styles.padding}>
+              <Button
+                invert
+                onPress={() => navigation.navigate(ScreenName.SueList)}
+                title="Meldungen ansehen"
+              />
+            </Wrapper>
+          )}
+
+        {!!staticContentListTitle && (
+          <WrapperVertical
+            style={[
+              styles.noPaddingBottom,
+              !!sueReportListNavigationButton &&
+                sueReportListNavigationButton === LIST_NAVIGATION_BUTTON.TOP &&
+                styles.noPaddingTop
+            ]}
+          >
+            <SectionHeader title={staticContentListTitle} />
+          </WrapperVertical>
+        )}
+
+        {!!staticContentListDescription && (
+          <Wrapper>
+            <RegularText>{staticContentListDescription}</RegularText>
+          </Wrapper>
+        )}
+
+        <ListComponent
+          data={staticContentListItem}
+          horizontal={horizontal}
+          navigation={navigation}
+          query={QUERY_TYPES.STATIC_CONTENT_LIST}
+        />
+
+        {!!sueReportListNavigationButton &&
+          sueReportListNavigationButton === LIST_NAVIGATION_BUTTON.BOTTOM && (
+            <Wrapper noPaddingBottom>
+              <Button
+                invert
+                onPress={() => navigation.navigate(ScreenName.SueList)}
+                title="Meldungen ansehen"
+              />
+            </Wrapper>
+          )}
+      </ScrollView>
+    </SafeAreaViewFlex>
+  );
+};
+
+const styles = StyleSheet.create({
+  noPaddingBottom: {
+    paddingBottom: 0
+  },
+  noPaddingTop: {
+    paddingTop: 0
+  },
+  padding: {
+    paddingTop: normalize(22)
+  }
```

Comment:
please name this `paddingTop` or add the `paddingBottom: 0` here as well to fullfill the naming `padding` and have `<Wrapper style={styles.padding}>`

Reply:
why do you prefer this over my second suggestion?

Reply:
Both were options. I think I chose the first one, for no specific reason. ☺️